content/blog/2016/2016-04-11-gsoc2016-mentors-call.adoc

```diff
@@ -0,0 +1,40 @@
+---
+layout: post
+title: "Google Summer of Code. Call for Mentors"
+tags:
+- general
+- jenkins2
+- gsoc
+author: oleg_nenashev
+---
+
+As you probably know, Jenkins project has been accepted to
+link:https://developers.google.com/open-source/gsoc/timeline[Google Summer of Code 2016]. 
+
+During last month we were working with students in order to discuss their project ideas and to review their application drafts.
+Thanks again to all students and mentors for your hard work during about ten office hours and dozens of other calls/chats!
+
+### Current status
```

Comment:
It looks like AsciiDoctor  will treat these as headers [despite its documentation not mentioning it](http://asciidoctor.org/docs/user-manual/#doc-header)

I'm going to merge this but please use standard asciidoc headers in the future